treeherder/services/pulse/consumers.py

```diff
@@ -147,7 +143,7 @@ def get_bindings(self, queue_name):
 
 
 class TaskConsumer(PulseConsumer):
-    queue_suffix = env("PULSE_TASKS_QUEUE_NAME", default="tasks")
+    queue_suffix = env("PULSE_TASKS_QUEUE_NAME", default="tasks_{}".format(uuid.getnode()))
```

Comment:
Leave the `TaskConsumer` and `PushConsumer` untouched since those are the ones that production is using and they use the default values.